apps/tailwind-components/pages/DataFetch.other.vue

```diff
@@ -0,0 +1,123 @@
+<script setup lang="ts">
```

Comment:
ok, I might like this over the client as a pattern but there is much knowledge in the client that we are now duplicating.